frontend/packages/dev-console/src/components/pipelines/PipelineVisualizationTask.scss

```diff
@@ -46,55 +56,66 @@ $gutter: 1.8em;
   }
 
   &__status {
-    margin: 0 4px;
+    margin: 0 3px 0 7px;
     width: 1.5em;
+    height: 1.5em;
     flex-grow: 0;
     flex-shrink: 0;
     order: -1;
     display: flex;
     justify-content: center;
     align-items: center;
-    font-size: 1.1em;
   }
 
   &__title {
-    flex-grow: 1;
-    flex-shrink: 1;
+    flex: 0 0 75px;
     text-overflow: ellipsis;
     overflow: hidden;
     white-space: nowrap;
-    margin: 0px 4px;
     &:hover {
       cursor: pointer;
     }
     &.is-text-center {
```

Comment:
Oops, I may have forgot to add some extra styles I had for this. I think we just visually broke the pills on the pipeline overview screen.

```
      flex-basis: 0; // get rid of the 75px width restriction
      flex-grow: 1; // fill the space it needs (up to a 100px)
```